x/treasury/spec/05_events.md

```diff
@@ -0,0 +1,26 @@
+# Events
+
+The bonds module emits the following events:
```

Comment:
This mentions bonds but we're not dealing with bonds here.

Reply:
changed